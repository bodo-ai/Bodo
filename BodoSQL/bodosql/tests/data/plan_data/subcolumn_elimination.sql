SELECT A as x,
    x + 10 as x2,
    x2 / 2 as x3,
    x3 * 3 as x4,
    B + (D - 17) * 7 as x5,
    D - 17 as x6
FROM table1
