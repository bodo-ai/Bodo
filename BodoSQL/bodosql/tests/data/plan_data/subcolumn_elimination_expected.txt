PandasProject(x=[$0], x2=[$1], x3=[$2], x4=[*($2, 3)], x5=[$3], x6=[$4])
  PandasProject(x=[$0], x2=[$1], x3=[/($1, 2)], x5=[+($3, *($2, 7))], x6=[$2])
    PandasProject(x=[$0], x2=[+($0, 10)], x6=[-($3, 17)], $f3=[$1])
      PandasTableScan(table=[[__bodolocal__, table1]])
